src/debiai/debiai.js

```diff
@@ -193,3 +193,15 @@ exports.selectionDataIdList = (req, res) => {
     }
 }
 
+exports.createSelection = (req, res) => {
+    try {
```

Comment:
Add a comment explaining that this is called when a selection is created from the DebiAI dashboard.
Precise that this an optional function and that throwing an error (if project doesn't support selection creation) will be supported by DebiAI